### DISCUSSION TITLE
Anzeige der Software-Version via MQTT

### DISCUSSION THREAD

**Discussion by Sprudelverduenner** [dewenni/ESP_Buderus_KM271#75] [General]
Hallo Buderus-Freunde,

gibt es eine Möglich per MQTT sich ein Topic mit der Version der installierten Firmware anzeigen zu lassen?
Ich benutze FHEM und hätte gerne dort ein dementsprechendes Reading...

Besten Dank vorab.
LG, Sprudelverduenner

**Comment by dewenni** [comment]
Hi @Sprudelverduenner 
die Version sollte jetzt schon per MQTT verfügbar sein.
Bis zur Version 3.3.1 gibt es eine /debug/ Message in der als JSON unter anderem die Version mit enthalten ist.

![image](https://github.com/dewenni/ESP_Buderus_KM271/assets/46074831/cf6468d1-292e-40ec-8326-93ce2b22ebcf)

Ich bin aber gerade bei diese zusätzlichen Informationen in der nächsten Version eventuell nochmal etwas anders zu gruppieren.
Die Versions-Info soll es aber weiterhin geben.

**Reply by Sprudelverduenner** [reply]
Hi dewenni, 
Danke für Deine Antwort.
Ja diese debug Message hatte ich bereits gesehen.

Ich wollte nur die Versionsinfo mir auf einer Webseite von FHEM anzeigen lassen. Wenn Du nun sagst, dass Du evtl noch ein einzelnes Topic generierst... dann warte ich noch.

Wenn Du es beim bisherigen Zustand belassen wolltest, dann werde ich mich damit beschäftigen müssen diesen json string irgendwie für meine Zwecke zu zerlegen...

**Reply by dewenni** [reply]
als einzelnes Topic wollte ich es eigentlich nicht machen.
Aber ganz sicher bin ich mir da aktuell auch noch nicht :-)

**Reply by Sprudelverduenner** [reply]
Habe es gerade schon mit doch sehr wenig aufwand hinbekommen den JSON STRING zu zerlegen.

Ich habe nun alles was ich wollte... wegen mir brauchst Du das nicht mehr zu implementieren.

Schönen Abend.

LG, Sprudelverduenner
